src/lib/FlightTasks/CMakeLists.txt

```diff
@@ -31,23 +31,87 @@
 #
 ############################################################################
 
-px4_add_library(flight_tasks
-tasks/FlightTask.cpp
-		tasks/FlightTaskManual.cpp
-		tasks/FlightTaskManualStabilized.cpp
-		tasks/FlightTaskOrbit.cpp
-		tasks/FlightTaskManualAltitude.cpp
-		tasks/FlightTaskManualAltitudeSmooth.cpp
-		tasks/FlightTaskManualPosition.cpp
-		tasks/FlightTaskManualPositionSmooth.cpp
-		tasks/FlightTaskAuto.cpp
-		tasks/FlightTaskAutoLine.cpp
-		tasks/FlightTaskAutoMapper.cpp
-		tasks/FlightTaskAutoFollowMe.cpp
-		tasks/FlightTaskOffboard.cpp
-		tasks/Utility/ManualSmoothingZ.cpp
-		tasks/Utility/ManualSmoothingXY.cpp
-		tasks/Utility/StraightLine.cpp
-		SubscriptionArray.cpp
-		FlightTasks.cpp
+###########################################
+# Prepare flight tasks
+###########################################
+
+# add upstream flight tasks (they are being handled differently from the core below)
```

Comment:
this comment is abit confusing, at least I don't exactly know what "differently" means.  
Am i correct with this statement:
Core FlightTasks cannot be removed. However, there are additional FlightTasks that are compiled by default (such as Orbit) but can be removed by adding the name of the FlighTasks to the `flight_tasks_to_remove` within a specific cmake target file.

Reply:
It's in a different list because the python script handles them differently because of `updateCommand()`

Reply:
And yes, you're right about your statement.

Reply:
I guess we will add that to the documentation